README.md

```diff
@@ -145,8 +138,6 @@ Your logo will show up here with a link to your website.
 <a href="https://opencollective.com/openjscad/sponsor/8/website" target="_blank"><img src="https://opencollective.com/openjscad/sponsor/8/avatar.svg"></a>
 <a href="https://opencollective.com/openjscad/sponsor/9/website" target="_blank"><img src="https://opencollective.com/openjscad/sponsor/9/avatar.svg"></a>
 
-HUGE THANKS and SHOUTOUT to [nodeBB](https://nodebb.org/) for hosting our (awesome) forum for free ! Thanks a lot !
```

Comment:
Why remove this ?

Reply:
I think this was repeated several times, so I removed the extra inserts. I’ll double check.

Reply:
Contributors was added back.